src/main/kotlin/no/nav/amt/arena/acl/metrics/DeltakerMetricHandler.kt

```diff
@@ -12,6 +12,18 @@ class DeltakerMetricHandler(
 ) {
 
 	fun publishMetrics(message: ArenaDeltakerKafkaMessage) {
+
+		// Flyttet fra et annet sted, er noe rart her men må undersøke hva denne brukes til
```

Comment:
Har sett gjennom de grafanaboardene jeg ser for meg kan være relevante, og jeg kan ikke se at den brukes 🤔